scripts/stanncam/stanncam.gml

```diff
@@ -1,11 +1,11 @@
-/// @function stanncam()
-/// @param x pos
-/// @param y pos
-/// @param width
-/// @param height
-/// @param surface_extra_on_ use surface_extra in regular draw events
-/// @param smooth_draw_ use fractional camera position when drawing
+/// @constructor stanncam
 /// @description creates a new stanncam
+/// @param {Real} [x_=0] - X position
+/// @param {Real} [y_=0] - Y position
+/// @param {Real} [width_=global.game_w]
+/// @param {Real} [height_=global.game_h]
+/// @param {Bool} [surface_extra_on_=false] - use surface_extra in regular draw events
+/// @param {Bool} [smooth_draw_=true] - use fractional camera position when drawing
```

Comment:
I'm now implementing stanncam into a new project, and i find these [x_=0] a bit distracting in practice.
Are they a feather standard?

I think it might be better to remove them again, since they clutter this space up a lot
![GameMaker_97c9NlCujy](https://user-images.githubusercontent.com/46312671/224566091-72a8da54-6c88-499a-a135-24a622d43183.png)

Reply:
Yeah, that doesn't look right. What version of GameMaker are you using?

Poke around the `Preferences > Feather Settings > Compact tooltip` to see if that improves it.

This is what my LTS looks like:

![GameMaker-LTS_OPDd2NjOQd](https://user-images.githubusercontent.com/20096702/224568047-63475869-196e-4101-9022-4d7e02e00e4c.png)